flake.nix

```diff
@@ -0,0 +1,57 @@
+{
+  description = "tkcli devshell";
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-22.11";
+    #nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable"; # localstack is broken right now (2023-01-25) in unstable, due to a missing dependency
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, ... }@inputs:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+
+        gci = pkgs.buildGoModule rec {
+           name = "gci";
+           src = pkgs.fetchFromGitHub {
+              owner = "daixiang0";
+              repo = "gci";
+              rev = "v0.10.1";
+              sha256 = "sha256-/YR61lovuYw+GEeXIgvyPbesz2epmQVmSLWjWwKT4Ag=";
+           };
+           #vendorSha256 = pkgs.lib.fakeSha256;
```

Comment:
Kind of: it's useful for setting that when upgrading, because it makes it clear which hash is intended, and I always forget the name.  That's why I keep it commented here.

Reply:
I see. Then maybe an extra comment above this can help?

```
# Uncomment this line if you need to upgrade
#vendorSha256 = pkgs.lib.fakeSha256;
```